nvidia-bert/README.md

```diff
@@ -173,14 +166,21 @@ Note that the datasets used for BERT pre-training need a large amount of disk sp
 
     Consult [Parameters](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/LanguageModeling/BERT#parameters) section by NVIDIA for additional details.
 
+6. Launch interactive container.
+
+    ```bash
+    cd workspace/BERT
+    bash ../../nvidia-bert/docker/launch.sh
+    ```
+
 7. Launch pre-training run
 
     ```bash
-    bash /workspace/scripts/run_pretraining_ort.sh
+    bash /workspace/bert/scripts/run_pretraining_ort.sh
```

Comment:
Let's add "set -e" to the shell script so that phase 2 does not try to run if phase 1 call fails? The fact that phase 2 runs when phase 1 fails is confusing at first glance. It takes people staring at the shell script for a few minutes or more to figure out what's happening. Long-term it would be better to change the approach of invoking both phases from one script invocation - it's unreliable.